geodiff/CMakeLists.txt

```diff
@@ -211,24 +182,10 @@ IF (BUILD_TOOLS)
 ENDIF (BUILD_TOOLS)
 
 ADD_DEFINITIONS( -DSQLITE_CORE )
```

Comment:
is this one required still ?

Reply:
apparently yes, otherwise the compilation fails (both shared and static lib)